### ISSUE TITLE
Database table civicrm_api_call: change type of column 'reply'

### ISSUE THREAD

**Issue by arnalyse** [CiviMRF/cmrf_core#54]
Hi there 👋

We ran in into an issue where the response of a Civi API call couldn't be stored in the `civicrm_api_call` table's `reply` column, because it was too long. 

The php error was: 

```
mysqli_sql_exception: Data too long for column 'reply' at row 1 in mysqli_stmt->execute() 
(Line 184 in /vendor/civimrf/cmrf_abstract_core/CMRF/PersistenceLayer/SQLPersistingCallFactory.php).
```

We were able to fix this by setting the column type to `longtext`, and would like to discuss if this would make sense in general as well.

**Comment by jensschuppe** [comment]
Sounds like a good idea to me - do you mind providing a PR for this, @arnalyse?

**Comment by arnalyse** [comment]
Yep. Will do, but may take a while. Time before Christmas is rare… 😆 🎅 🎁
